packages/db/prisma/seed/web3.ts

```diff
@@ -18,7 +18,7 @@ export const COMMON_MAX_FEE_PER_BLOB_GAS = [
   14200000000, 40000000000, 56649567354, 52371487834, 54468542552, 44760150790,
 ].map((x) => BigInt(x));
 
-export const ROLLUP_ADDRESSES: Record<Rollup, string> = {
+export const ROLLUP_ADDRESSES: Partial<Record<Rollup, string>> = {
```

Comment:
```suggestion
export const ROLLUP_ADDRESSES = {
```
Removes redundant type declaration since it can be inferred from the object literal:
<img width="779" alt="image" src="https://github.com/user-attachments/assets/579d8bfd-4228-404a-b448-c8d0173ac6c0" />